_posts/en/2021-06-08-working-in-the-open-part-2.md

```diff
@@ -0,0 +1,172 @@
+
+## Inner Source vs Open Source
+
+It's worth reminding ourselves that not everyone is mentally and emotionally ready to open up to the whole world.
+Various reasons exist and we should not judge.
+
+As mentioned in our [previous post]({{site.baseurl}}{% post_url 2019-11-19-working-in-the-open-part-1 %}), some organizations have taken the "Inner Source" road, opening up all workspaces to all their internal teams, to help folks slowly get comfortable with working in the open as well as reaping part of its benefits.
+Doing so can help identify what works well and what challenges surfaces for your team and your organization.
+It can also help figure out what mechanisms of control could be added to ensure that the organization keeps a good hold on its information and how to better support its employees in adopting the best practices.
+
+If your team isn't fully ready to work in the open, try slowly opening up your workspace to the rest of the organization by default.
+Don't wait until "that presentation is ready" before letting it be discoverable by others.
+You know you will not get around to it!
+
+## Set up your Environment
+
+Your virtual workspace may consist of a variety of tools and platforms.
+
+Software developers and IT folks may leverage social coding platforms such as Gitlab, Github, GCcode, etc., to collaborate widely with people around the world on source code.
+
+In the Government of Canada, most departments now use Microsoft 365 to varying degrees, which may include Teams, a collaboration platform that ties in multiple products and capabilities.
+
+Whether you are using the corporate collaboration platform or a 3rd party software as service to work in the open, there are a number of things to keep in mind.
+
+### Stay in Control
+
+Your workspace is a very important place: it contains your work and you have a responsibility to ensure its properly secured and controlled.
+However, that doesn't mean you can't open up securely.
+
+Proper access management is required so that we can separate the two main types of users: your team's members and external contributors.
+The latter could be a person from within your organization or from the public.
+
+There may be more types of roles available depending on the platforms, but these are the main buckets we think of when working in the open.
+
+Depending on the platform you use, various controls may be offered to let you openly share your documents and your work while remaining in control of its content.
+
+In SharePoint, a platform most often available as "Inner Source" only, a team can open up their workspace by default to their colleagues and create specific folders with more restricted access to privately work on documents only when absolutely required.
+
+With most collaboration platforms, whether it's SharePoint, Google Documents, etc., there usually is an option to let external contributors consult your documents and add comments but prevent them from making changes directly.
```

Comment:
Pretty sure the official name is Google Docs not Google Documents (since we are talking about the platform)

Reply:
Ha! Yes, good point :smile: